include/ranges.h

```diff
@@ -223,11 +223,10 @@ id<DimsOut> id_cast(const coordinate<InterfaceIn, DimsIn>& in) {
 	return coordinate_cast<id<DimsOut>>(in);
 }
 
-struct zero_range_t {
-} inline static constexpr zero_range;
-struct unit_range_t {
-} inline static constexpr unit_range;
-
+struct zero_t {
+} inline static constexpr zero;
+struct one_t {
+} inline static constexpr one;
```

Comment:
I'm not entirely happy with this renaming. Without knowing the definition one could assume that `id(one)` refers to `[0,0,1]` instead of `[1,1,1]`. `zero` is unambiguous in that regard but from the name alone it's unclear that it is only used to construct ranges or ids.

How about `zero_coord` and `unit_coord`, or at least `zeros` and `ones`?

Reply:
I like `zeros` and `ones`, I'll do that.

Reply:
Done.